.github/workflows/release.yml

```diff
@@ -27,7 +27,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
           NPM_TOKEN: ${{secrets.NPM_TOKEN}}
-          SLACK_WEBHOOK: ${{secrets.SLACK_WEBHOOK}}
```

Comment:
@kevinbuhmann did you remove the `SLACK_WEBHOOK` from the secrets as well?

Reply:
I will.